src/dodal/devices/s4_slit_gaps.py

```diff
@@ -1,8 +1,58 @@
-from ophyd import Component, Device, EpicsMotor
+from ophyd_async.core import DeviceVector, StandardReadable
+from ophyd_async.epics.motion import Motor
 
 
-class S4SlitGaps(Device):
-    """Note that the S4 slits have a different PV fromat to other beamline slits"""
+class S4SlitGaps(StandardReadable):
```

Comment:
Because I don't know what S4 is :)

Will rename to `Slits`

Reply:
Incidentally, and at risk of betraying my woeful knowledge of xray optics, if either you or @Tom-Willemsen can come up with a better docstring than mine then I'm all ears. Mine is basically "erm, it's a set of some sort of slits...?"

Reply:
This is a 4-blade slit set defined by a gap and a centre.

The things I cam immediately think of which we want to diambiguate from are:
- Having different numbers of blades, e.g. a 2 blade vertical slit set might exist on some beamlines, this would only define the beam in a vertical oritentation.
- Definining the slit set by absolute positions of each blade rather than gap and centre. Not _sure_, but I'd wager at least some beamlines do this - e.g. for scanning in each blade against the beam individually

Reply:
Ah, so could actually call the class `Slits4Blade` or something similar? For absolute vs. relative it sounds like those things can coexist in the same device?

I'd love to swing by but I'm sick!

Reply:
We can probably assume 4-blade is the standard case, so I think naming the class `Slits` is fine as long as the docstring clarifies it's a 4-blade set.

> For absolute vs. relative it sounds like those things can coexist in the same device?

Yes, care is just needed because you then have multiple PVs which affect the same physical blade. (e.g. moving the north blade will change the vertical gap and vice versa). 

Does a picture like this help to visualise what's going on?

![image](https://github.com/DiamondLightSource/dodal/assets/24375813/9e527402-ef06-49f9-85d6-bcd912fc1033)

Reply:
Yes, quite a lot, thanks!